src/cf-forms/src/atoms/multiselect.less

```diff
@@ -0,0 +1,16 @@
+.a-multiselect {
+    display: block;
+
+    box-sizing: border-box;
+    width: 100%;
+    height: 5.5em; // This height breaks the bottom border mid-character so it's clear there's more
```

Comment:
I was thinking in case users ever wanted to easily override the default, but it's not a huge deal.